Makefile

```diff
@@ -313,7 +313,22 @@ test: ##@test Run tests once in NodeJS
 	yarn shadow-cljs compile test && \
 	node --require ./test-resources/override.js target/test/test.js
 
-#--------------
+component-test-watch: export TARGET := clojure
+component-test: export __COMPONENT_TEST__ := true
+component-test: export BABEL_ENV := test
+component-test-watch: ##@ Watch tests and re-run no changes to cljs files
```

Comment:
I find the feedback cycle of the watcher a bit too slow (not much we can do anyway), but have you tried running the tests in a REPL?

Reply:
I have not tried it in a REPL yet, it's something we should definitely explore

Reply:
Cool, good to know :+1: